config/prometheus/README.md

```diff
@@ -29,3 +29,20 @@ The files [install-prometheus.yaml](install-prometheus.yaml) and
 [servicemonitors.yaml](servicemonitors.yaml) are taken from
 [istio/tools](https://github.com/istio/tools/), with configuration related to
 Istio left out.
+
+### Prometheus scrape interval
+
+The scrape interval (`scrape_interval`) determines how often Prometheus scrapes
+targets. The default value for this interval is
+[1m](https://prometheus.io/docs/prometheus/latest/configuration/configuration/).
+We change this interval in our configuration to 1s, since our tests typically
+last only 30s. See <https://github.com/grpc/test-infra/pull/325> for details.
+
+### Delay to query
+
+We add 20s delay before collecting data from Prometheus, in the tests that use
+Prometheus. This delay allows data to become available
+([cAdvisor housekeeping interval](https://github.com/google/cadvisor/blob/master/docs/runtime_options.md#housekeeping))
+and pulled by Prometheus
+([Prometheus scrape interval](https://github.com/grpc/test-infra/pull/325)). See
+details in <https://github.com/grpc/test-infra/pull/330>.
```

Comment:
I would say "See <> for details." because this is the same form that you used in the previous section.

Reply:
Sorry I missed it.